gobblin-service/src/main/java/org/apache/gobblin/service/modules/orchestration/NewDagManager.java

```diff
@@ -283,9 +283,10 @@ public void handleKillFlowEvent(KillFlowEvent killFlowEvent) {
     }
   }
 
-  //@Subscribe todo uncomment it when new dag manager is ready for resume functionality
+  @Subscribe
   public void handleResumeFlowEvent(ResumeFlowEvent resumeFlowEvent) {
     if (!this.dagProcessingEngineEnabled) {
+      log.info("Not handling resume flow event yet.");
```

Comment:
throw unsupported exception here or make this a warn/error level so it doesn't escape notice